app/assets/stylesheets/arclight/modules/context_navigation.scss

```diff
@@ -32,6 +32,7 @@ li.al-collection-context  {
 
   .al-document-abstract-or-scope {
     max-width: 45em;
+    margin-bottom: 0;
```

Comment:
Could/should we use the `mb-0` class on the element here instead?

Reply:
Oh good idea. I've pushed up this change!